how-to/run-kata-with-k8s.md

```diff
@@ -112,10 +112,15 @@ default_workload_trust = "untrusted"
 
 #### Network namespace management
 To enable networking for the workloads run by Kata, CRI-O needs to be configured to
-manage network namespaces, by setting the following key to `true`:
+manage network namespaces, by setting the following key to `true` in CRI-O v1.16:
```

Comment:
Thanks for fixing this @cmaf. Just a small nit: Can you place the "In CRI-O v1.16" on a new line?

Reply:
@amshinde yes, just updated it.